gui/index.html

```diff
@@ -63,80 +56,82 @@
 			<a class="nav-link text-light active" id="v-pills-home-tab" data-toggle="pill" href="#v-pills-home" role="tab"
 				aria-controls="v-pills-home" aria-selected="true">Home</a>
 			<a class="nav-link text-light" id="v-pills-options-tab" data-toggle="pill" href="#v-pills-options" role="tab"
-			aria-controls="v-pills-options" aria-selected="false">Options</a>
+				aria-controls="v-pills-options" aria-selected="false">Options</a>
 		</div>
 		<!-- End Sidebar -->
 
 		<!-- Page Content -->
-		<div class="flex-grow-1">
-			<div class="tab-content container p-2 col-lg-10" id="v-pills-tabContent">
-				<!-- v-pills-home -->
-				<div class="tab-pane fade show active" id="v-pills-home" role="tabpanel" aria-labelledby="v-pills-home-tab">
-					<div class="d-flex float-right">
-						<button id="save" class="btn btn-primary" onclick="ahk.sb_saveCurrent()">Save</button>
-						<button id="open" class="btn btn-secondary" onclick="ahk.sb_openBTN()">Open</button>
-					</div>
-					<h2 id="mainDate"></h2>
-					<form onsubmit="ahk.sb_saveCurrent()">
-						<h2 id="title" class="font-weight-bold"></h2>
+		<div class="flex-grow-1 d-flex flex-column">
+			<div class="flex-grow-1">
```

Comment:
If you want the footer to always be at the bottom of the page, then you need to scroll the contents of this div:

`style="overflow-y:auto"` to ensure just this section overflows.